Src/WitsmlExplorer.Frontend/models/rig.tsx

```diff
@@ -1,31 +1,57 @@
+import Measure from "./measure";
+
 export default interface Rig {
-  airGap: string;
+  airGap: Measure;
```

Comment:
Inconsistent use of letter casing. Use lowercase.

Reply:
I believe it has to be as it is, else it will be troublesome to load the xml document properly